Libs/RaptureCore/src/main/java/rapture/mail/SMTPConfig.java

```diff
@@ -34,39 +34,50 @@ public String getUsername() {
         return username;
     }
 
-    public void setUsername(String username) {
+    @Override
+    public String toString() {
+        return "SMTPConfig [host=" + host + ", port=" + port + ", username=" + username + ", password=" + password.replaceAll(".", "*") + ", from=" + from
+                + "]";
+    }
+
+    public SMTPConfig setUsername(String username) {
         this.username = username;
+        return this;
     }
 
     public String getPassword() {
         return password;
     }
 
-    public void setPassword(String password) {
+    public SMTPConfig setPassword(String password) {
```

Comment:
Let's you do daisy chaining (AKA decorator pattern):
add(new SMTPConfig().setPassword(foo).setUser(bar).setPort(1));
I'd like to change the Antlr code to do that to all the auto-generated stuff